android/jni/app-android.cpp

```diff
@@ -29,7 +36,351 @@
 
 #include "app-android.h"
 
-static JNIEnv *jniEnvUI;
+// HACK: Use NativeQueryConfig
```

Comment:
I'm gonna move this change back to the separate native later, so I do want to do it that way.